src/tasks/platforms/platform.ts

```diff
@@ -22,18 +22,33 @@ import { OpenshiftTasks } from './openshift'
 
 /**
  * Platform specific tasks.
- *  - preflightCheck
  */
 export class PlatformTasks {
-  preflightCheckTasks(flags: any, command: Command): ReadonlyArray<Listr.ListrTask> {
-    const minikubeTasks = new MinikubeTasks()
-    const microk8sTasks = new MicroK8sTasks()
-    const minishiftTasks = new MinishiftTasks()
-    const openshiftTasks = new OpenshiftTasks()
-    const k8sTasks = new K8sTasks()
-    const crc = new CRCHelper()
-    const dockerDesktopTasks = new DockerDesktopTasks(flags)
+  protected minikubeTasks: MinikubeTasks
+
+  protected microk8sTasks: MicroK8sTasks
+
+  protected minishiftTasks: MinishiftTasks
+
+  protected openshiftTasks: OpenshiftTasks
+
+  protected k8sTasks: K8sTasks
+
+  protected crc: CRCHelper
+
+  protected dockerDesktopTasks: DockerDesktopTasks
 
+  constructor(flags: any) {
+    this.minikubeTasks = new MinikubeTasks()
+    this.microk8sTasks = new MicroK8sTasks()
+    this.minishiftTasks = new MinishiftTasks()
+    this.openshiftTasks = new OpenshiftTasks()
+    this.k8sTasks = new K8sTasks()
+    this.crc = new CRCHelper()
+    this.dockerDesktopTasks = new DockerDesktopTasks(flags)
```

Comment:
Why do we need to create all these tasks if we going to use only one list?

Reply:
This tasks are moved to constructor not to duplicate in every function.

Reply:
It changed nothing for now, but ok. The refactoring is probably out of scope of this PR